config/environments/production.rb

```diff
@@ -115,13 +115,14 @@
 
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
+  sendgrid_api_key_present = ENV["SENDGRID_API_KEY"].present?
   config.action_mailer.default_url_options = { host: protocol + config.app_domain }
   ActionMailer::Base.smtp_settings = {
     address: "smtp.sendgrid.net",
     port: "587",
     authentication: :plain,
-    user_name: ENV["SENDGRID_USERNAME_ACCEL"],
-    password: ENV["SENDGRID_PASSWORD_ACCEL"],
+    user_name: sendgrid_api_key_present ? "apikey" : ENV["SENDGRID_USERNAME_ACCEL"],
```

Comment:
Should this _actually_ be `"apikey"`?

Reply:
(Just want to confirm because [the docs](https://sendgrid.com/docs/for-developers/sending-email/rubyonrails/#configure-actionmailer-to-use-sendgrid) make it seem like you're supposed to replace the values with your keys/info:
```
:password => 'your_sendgrid_api_key',
```

Reply:
I did it just the way the docs do
```
  :user_name => 'apikey',
  :password => 'your_sendgrid_api_key',
```

Reply:
Welp, those docs are confusing to me, but seems fine to try this out